components/display/magtag/definition.json

```diff
@@ -0,0 +1,17 @@
+{
+    "displayName": "MagTag E-Ink Display",
+    "vendor": "Adafruit",
+    "productURL": "https://www.adafruit.com/product/4800",
+    "documentationURL": "https://learn.adafruit.com/adafruit-magtag",
+    "published": false,
+    "description": "A 2.9\" black and white e-ink display onboard the Adafruit MagTag",
+    "displayType": {
+        "type": "epd",
+        "spiEpd": {
+            "bus": 0
+        },
+        "epdConfig": {
+            "mode": "mono"
```

Comment:
Think this should be greyscale4 not mono

Reply:
I do not understand - `epdConfig` offers two modes - `mono` and `grayscale4`.